client/src/components/review.jsx

```diff
@@ -4,47 +4,41 @@ import ReviewText from './reviewText';
 import Report from './report';
 import HostResponse from './hostResponse';
 
-class Review extends React.Component {
-  render() {
-    const month = {
-      '01': 'January',
-      '02': 'February',
-      '03': 'March',
-      '04': 'April',
-      '05': 'May',
-      '06': 'June',
-      '07': 'July',
-      '08': 'August',
-      '09': 'September',
-      '10': 'October',
-      '11': 'November',
-      '12': 'December',
-    };
-    const {
-      review,
-    } = this.props;
+const Review = ({ review }) => {
+  const month = {
+    '01': 'January',
+    '02': 'February',
+    '03': 'March',
+    '04': 'April',
+    '05': 'May',
+    '06': 'June',
+    '07': 'July',
+    '08': 'August',
+    '09': 'September',
+    '10': 'October',
+    '11': 'November',
+    '12': 'December',
+  };
+  const date = review.date.split('/');
+  const formatedDate = `${month[date[1]]} ${date[0]}`;
+  const hostResponse = review.id % 10 ? '' : <HostResponse date={formatedDate} hostResponse={review} />;
```

Comment:
Interesting, instead of explicitly stating review.id % 10 === 0, you have review.id % 10. That works.